miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/chat/ChatBridgeDispatcher.kt

```diff
@@ -18,4 +18,31 @@ interface ChatBridgeDispatcher {
         onSuccess: (contactId: String?) -> Unit,
         onError: (message: String) -> Unit
     )
+
+    /**
+     * Triggered when Mini App wants to send a message to a specific contact.
+     * Should send a message to the specified contactId without any prompt to the User.
+     * Should invoke [onSuccess] after message was successfully sent.
+     * Should invoke [onError] when there was an error.
+     */
+    fun sendMessageToContactId(
+        contactId: String,
+        message: MessageToContact,
+        onSuccess: (contactId: String?) -> Unit,
+        onError: (message: String) -> Unit
+    )
+
+    /**
+     * Triggered when Mini App wants to send a message to multiple contacts.
+     * Should open a contact chooser which allows the user to choose multiple contacts,
+     * and should then send the message to all chosen contacts.
+     * Should invoke [onSuccess] with a list of IDs of the contacts which were successfully sent the message.
+     * If the user cancelled sending the message, should invoked [onSuccess] with an empty list.
```

Comment:
@Climbatize @khairul-alam-licon @minh-rakuten Is this correct to return an empty list when cancelled? I thought we had discussed returning null in that case.

Reply:
yep, I changed this on iOS side, now I return `null`

Reply:
Yeah, I have updated this part.